src/main/plugin/iso19139.ca.HNAP/loc/eng/schematron-rules-common.xml

```diff
@@ -41,6 +41,8 @@
   <DistributionFormatVersion>Value is required for distribution format - File version</DistributionFormatVersion>
   <DistributionFormatInvalid>Value for distribution format is not valid</DistributionFormatInvalid>
   <OnlineResourceUrl>Value is required for Online Resource URL</OnlineResourceUrl>
+  <OnlineResourceName>Protocol of Online Resource "</OnlineResourceName>
+  <OnlineResourceProtocol>" is not valid. Valid values are: </OnlineResourceProtocol>
```

Comment:
I'm not a fan of splitting the message into to parts. It makes the second part confusing.
I think it would be better to use a placeholder

i.e. Protocol for Online Resource "{invalidProtocol}" is not valid. Valid values are: 

The placeholder can be anything as long as it is not text that could appear in the message.

And then search and replace the placeholder.  This makes it easier to upgrade the text in the future and it also makes it easier to understand/read the full message.